src/modules/vendor/components/programs/cards/templates/QuestionAnswer.vue

```diff
@@ -60,6 +63,10 @@ export default {
       return this.card.questionAnswers.length >= QUESTION_ANSWER_MAX_ANSWERS_COUNT ||
         this.disableEdition || this.activity.status === PUBLISHED;
     },
+    disableAnswerDeletion () {
+      return this.card.questionAnswers.length <= QUESTION_ANSWER_MIN_ANSWERS_COUNT ||
+        this.disableEdition || this.activity.status === PUBLISHED;
```

Comment:
Dans disableEdition, il n'y a pas déjà l'information de `this.activity.status === PUBLISHED` ?

Reply:
Je me base sur le isEditionLocked de ActivityProfile

Reply:
Normalement non parce que tu peux deverouiller une activité pour pouvoir modifier un texte par exemple. Ici meme si on deverouille la carte on ne veut pas pouvoir supprimer de reponse.